### ISSUE TITLE
Different output from cmark

### ISSUE THREAD

**Issue by ttakuru88** [kivikakk/comrak#316]
Hi. I've noticed that comrak behaves different from cmark.
Which behavior is correct?

### Markdown
```md
<span style="color:red;">**(ABC)**</span>
```

### cmark 0.30.3
```html
<p><!-- raw HTML omitted --><strong>(ABC)</strong><!-- raw HTML omitted --></p>
```

### comrak 0.18.0
```html
<p><!-- raw HTML omitted -->**(ABC)**<!-- raw HTML omitted --></p>
```

I hope cmark is correct.

- - -

Similar pattern.

### Markdown
```md
**(1)**=
```

### cmark
```html
<p><strong>(1)</strong>=</p>
```

### comrak
```html
<p>**(1)**=</p>
```

**Comment by kivikakk** [comment]
cmark is indeed correct.

**Comment by kivikakk** [comment]
This was fixed by #428!
